### PR TITLE
US150404 - Migrate get-lms-release to composite action

### PR DESCRIPTION
**Pull request by svanherk**
This brings the `get-lms-release` action more inline with the others in this repo, and we no longer need to hardcode a node version or check in `node_modules`. I separated this into two commits so it's hopefully easier to review - the first one is the actual changes and the second deletes the `node_modules` folder.

I've been testing this and comparing the old output with the new output in a test BSI PR: https://github.com/Brightspace/brightspace-integration/pull/8766/files
Example output: https://github.com/Brightspace/brightspace-integration/actions/runs/4554318642/jobs/8032013535

### CODE REVIEW

**Comment by svanherk** [comment]
> I separated this into two commits so it's hopefully easier to review - the first one is the actual changes and the second deletes the node_modules folder.

Or if it'd be easier, I can revert the second commit for now and then add it back before merging.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "lms-version-actions",
+    "version": "1.0.0",
+    "description": "Collection of GitHub Actions to help coordinate with LMS versioning",
+    "type": "module",
```
Comment (svanherk) [inline package.json]: This was needed to use nested JS files without having to change the extension.

**Review by dlockhart — APPROVED** [review]
Awesome! LGTM.

**Review by Scy-D2L — APPROVED** [review]
Nice! Much cleaner. :shipit:

**Pull request merged**